File: 백준/Bronze/26145. 출제비 재분배/출제비 재분배.py
import sys
input=lambda:sys.stdin.readline().rstrip()
n,m=map(int,input().split())
s=[*map(int,input().split())]
rlt=s+[0]*m
for i in range(n):
    mlist=[*map(int,input().split())]
    for j in range(n+m):
        rlt[i]-=mlist[j];
        rlt[j]+=mlist[j]
for i in rlt:
    print(i,end=' ')

File: 백준/Bronze/26145. 출제비 재분배/출제비 재분배.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int n, m;
    cin >> n >> m;
    vector<int> pay(n), rlt(n + m), mlist(n + m);
    for (int i = 0; i < n; i++) {
        cin >> pay[i];
        rlt[i] = pay[i];
    }
    for (int i = n; i < n + m; i++)
        rlt[i] = 0;
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n + m; j++) {
            cin >> mlist[j];
            rlt[i] -= mlist[j];
            rlt[j] += mlist[j];
        }
    }
    for (int i = 0; i < n + m; i++)
        cout << rlt[i] << ' ';
    return 0;
}